packages/vite-app-ts/.env

```diff
@@ -0,0 +1,14 @@
+VITE_APP_TARGET_NETWORK=localhost
+HARDHAT_TARGET_NETWORK=localhost
+
+# network
```

Comment:
Just double-checking, is all of this safe to be here?

Reply:
yeah, its all stuff austin pays for and is ok with other people working with it.  It was all hardcoded before.